code/modules/client/client_procs.dm

```diff
@@ -1193,18 +1193,14 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 	fullscreen = !fullscreen
 
 	if (fullscreen)
-		winset(usr, "mainwindow", "on-size=")
-		winset(usr, "mainwindow", "titlebar=false")
-		winset(usr, "mainwindow", "can-resize=false")
-		winset(usr, "mainwindow", "menu=")
-		winset(usr, "mainwindow", "is-maximized=false")
-		winset(usr, "mainwindow", "is-maximized=true")
+		// ATTN!! ONCE 515.1631 IS REQUIRED REPLACE WITH winset(src, "mainwindow", "menu=;is-fullscreen=true") (and remember to replace the other call of course)
+		// this means no double maximise calls to make sure window fits, and supresses titlebar, can-resize and is-maximized here making those redundant
+		// both implementations are functionally "windowed borderless"
```

Comment:
If we just leave it a comment we'll never remember lol. Can you use something like...
```c
#if (MIN_COMPILER_VERSION > 515 || MIN_COMPILER_BUILD > 1630)
#warn [your descriptive and awesome warning here]
#endif
```